examples/src/examples/graphics/ground-fog.tsx

```diff
@@ -256,8 +256,9 @@ class GroundFogExample {
 
                 let firstFrame = true;
                 let currentTime = 0;
+                let c = 0;
                 app.on("update", function (dt) {
-
+                    c++;
```

Comment:
I did a double-take here. C++? This is JavaScript!

Reply:
right? sorry removing changes to this file, c++ just won't do it.